spec/services/braintree_service/escrow_release_helper_spec.rb

```diff
@@ -50,17 +50,22 @@
   end
 
   describe "#next_escrow_release_time" do
+
+    def central_time(*args)
+      ActiveSupport::TimeZone.new("Central Time (US & Canada)").local(*args)
+    end
+
     context 'when todays batch has not been run yet' do
       it 'returns time of todays batch and buffer' do
-        next_batch = BraintreeService::EscrowReleaseHelper.next_escrow_release_time(Time.new(2014, 3, 11, 22, 0, 0, 0), 2)
-        next_batch.utc.should be_eql(Time.new(2014, 3, 12, 1, 0, 0, 0))
```

Comment:
Central Time is GMT-6 without daytime saving and GMT-5 with it. Thus, this line will always fail in summertime. It's better to compare it directly to desired 19:00 in Central Time. Also, 22:00 UTC should resolve to the next day in winter (because 22-5 is 17 and current day's batch is already passed) and to the current day in summer (22-6 is 16, the batch is in an hour). This is why I changed it to 21:00, which should always resolve to current day's batch.